packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -425,7 +425,7 @@ impl PostgresDatabase {
         transaction: &mut Transaction<'_, Postgres>,
         entity_type: &Qualified<EntityType>,
     ) -> Result<(), InsertionError> {
-        // TODO: Store this as mapping in `entity_type_property_types`
+        // TODO: Store this as mapping in `entity_type_property_type_references`
```

Comment:
Perhaps an oversight on the other PR, but is this TODO relevant anymore? I think we are inserting into the table now

Also on this line https://github.com/hashintel/hash/blob/07dc1161929b9fc873f700b0d1485819fa451cbc/packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs#L442 we should rename the table I believe

Reply:
Ah I think my rebase messed up (and the rebase was accidental, I should have merged). Thank you, will fix

Reply:
The integration tests passed locally though which makes me wonder why the rename didn't cause problems. (Although it's failing in CI)